storage/client_split_test.go

```diff
@@ -506,15 +506,19 @@ func TestStoreRangeSystemSplits(t *testing.T) {
 	if err := store.DB().Txn(func(txn *client.Txn) error {
 		txn.SetSystemDBTrigger()
 		for _, kv := range sql.GetInitialSystemValues() {
-			if errPut := txn.Put(kv.Key, kv.Value.Bytes); errPut != nil {
-				return util.Errorf("Could not put KV %+v: %v", kv, errPut)
+			if err := txn.Put(kv.Key, kv.Value.Bytes); err != nil {
+				return err
 			}
 		}
 		for i := 1; i <= 5; i++ {
 			// We don't care about the values, just the keys.
 			k := sql.MakeDescMetadataKey(sql.ID(keys.MaxReservedDescID + i))
-			if errPut := txn.Put(k, i); errPut != nil {
-				return util.Errorf("Could not put KV %+v: %v", k, errPut)
+			v, err := txn.Get(k)
```

Comment:
This test was previously writing bogus values into the system DB entries. The new stricter unmarshalling code in gogoproto got grumpy, so I had to change this to write not-bogus values.